bin/run_tests.sh

```diff
@@ -0,0 +1,10 @@
+# This script is used to run tests in Meson
+script_path=`realpath "$0"`
+bin_dir=`dirname "$script_path"`
+base_dir=`dirname "$bin_dir"`
+
+export SHELL="$base_dir/build/src/cmd/ksh93/ksh"
+cd "$base_dir/src/cmd/ksh93/tests"
+
+# Skip shcomp tests for now
```

Comment:
It's not clear what this comment means. There is a *./src/cmd/ksh93/tests/shtests* script but no corresponding *shcomp* script. In any event it's generally better to write such coments as TODO's; e.g., `TODO: Add ./shcomp after we've implemented xyz.`. That way we can `grep TODO` to find things that need to be done.

Reply:
```shcomp``` tests are executed from within ```shtests``` script. I will update the comment to make it more clear.